src/App.vue

```diff
@@ -3,8 +3,9 @@
     <Header :loggedIn="loggedIn"/>
     <Login v-if="!loggedIn" v-on:newUser="createNewUser($event)" v-on:userLogin="updateLogin()"/>
     <main v-else>
-      <router-view />
-      <Home/>
+      <router-view 
```

Comment:
Glad we can pass props down. This will allow us to pull users from any component.